static/app/types/group.tsx

```diff
@@ -110,6 +111,54 @@ export enum IssueTitle {
   PERFORMANCE_LARGE_HTTP_PAYLOAD = 'Large HTTP payload',
   PERFORMANCE_HTTP_OVERHEAD = 'HTTP/1.1 Overhead',
   PERFORMANCE_DURATION_REGRESSION = 'Duration Regression',
+  PERFORMANCE_ENDPOINT_REGRESSION = 'Endpoint Regression',
+
+  // Profile
+  PROFILE_FILE_IO_MAIN_THREAD = 'File I/O on Main Thread',
+  PROFILE_IMAGE_DECODE_MAIN_THREAD = 'Image Decoding on Main Thread',
+  PROFILE_JSON_DECODE_MAIN_THREAD = 'JSON Decoding on Main Thread',
+  PROFILE_REGEX_MAIN_THREAD = 'Regex on Main Thread',
+  PROFILE_FRAME_DROP = 'Frame Drop',
+  PROFILE_FRAME_DROP_EXPERIMENTAL = 'Frame Drop',
+  PROFILE_FUNCTION_REGRESSION = 'Function Regression',
+  PROFILE_FUNCTION_REGRESSION_EXPERIMENTAL = 'Function Duration Regression (Experimental)',
+
+  // Replay
+  REPLAY_RAGE_CLICK = 'Rage Click Detected',
+}
+
+const ISSUE_TYPE_TO_ISSUE_TITLE = {
```

Comment:
is this the same thing as the enum above but in lower case?

Reply:
The keys for this dict? Yes they are. I couldn't find a non-janky way to match an IssueType value (like `performance_db_main_thread`) back to it's key (`PERFORMANCE_DB_MAIN_THREAD`) so we could get the IssueTitle enum with the same key (`DB on Main Thread`). Do you know if there's a better way to do this other than creating a separate (and kinda redundant) dict?

Reply:
Thanks that clears up my question 👍 added a different suggestion